### ISSUE TITLE
VSCode Extension ignoring stylua.toml

### ISSUE THREAD

**Issue by JoelBrd** [JohnnyMorganz/StyLua#915]
Began coding this morning and upon formatting my files, had all my spaces turn into tabs. Saw the VSCode extension got updated to 1.7.0 last night.

Reverting back to 1.6.3 resolves the issue and formatting resumes back to the settings in my `stylua.toml`

**Comment by JohnnyMorganz** [comment]
What is the structure of your workspace? (i.e. the location of the file relative to your stylua.toml and the folder you have open in VSCode)

**Comment by JoelBrd** [comment]
I have `folder` open

```
folder
  - stylua.toml
  - test.lua
  - src
    - client
      - Tiles
        - TileBoard.lua
```


I get space indentation in `test.lua`, but tab indentation in `src/client/Tiles/TileBoard.lua`


`stylua.toml`
```
column_width = 140
line_endings = "Windows"
indent_type = "Spaces"
indent_width = 4
quote_style = "AutoPreferDouble"
call_parentheses = "Always"
collapse_simple_statement = "Never"
```

**Comment by JohnnyMorganz** [comment]
Agh, thanks.

This is unfortunately me misunderstanding the way I originally implemented `stdin-filepath` and how it relates to config resolution.

I'm going to fix this properly in #831 which will be in the next release. But for now, you can either use the older version of rename "search parent directories" in the vscode settings
